src/java/org/apache/cassandra/exceptions/RequestValidationException.java

```diff
@@ -28,4 +28,12 @@ protected RequestValidationException(ExceptionCode code, String msg, Throwable e
     {
         super(code, msg, e);
     }
+
+    public static class UnsafeException extends RequestValidationException
```

Comment:
this is super ugly, but lets us handle every type... i can also make the base class not abstract...